openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -317,7 +328,10 @@ TCN_IMPLEMENT_CALL(jint, SSLContext, free)(TCN_STDARGS, jlong ctx)
     UNREFERENCED_STDARGS;
     TCN_ASSERT(ctx != 0);
 
-    return ssl_context_cleanup(c);
+    ssl_context_cleanup(c);
+
+    // TODO: Fix me
```

Comment:
Doing a followup PR is fine but the return value is not used in Netty. I don't think this will break anything.